<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmAppServeurTcp</class>
 <widget class="QMainWindow" name="CIhmAppServeurTcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>1002</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveur TCP de base</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="leTexte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>861</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teTexte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>1101</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEnvoyer">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>215</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lTexte">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>781</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Ecoute sur le port </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbListe">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>331</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Clients connect√©s : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Effacer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>44</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>teTexte</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
